### PR TITLE
[SCSS] Buttons & Textfields v2

### PR DESCRIPTION
**Pull request by jeremie-lucca**
image

image

image

image

image

image

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://358-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://358-pr-lucca-front-LuccaSA.surge.sh

File: packages/scss/src/theming/_palettes.scss [hunk 1]
```diff
@@ -5,44 +5,69 @@ $palettes: (
...
 		"transparent": rgba(#FF6600, .1),
 		"dark": darken(#FF6600, 5%),
 		"darker": darken(#FF6600, 20%),
-		"light": lighten(#FF6600, 5%),
-		"lighter": lighten(#FF6600, 20%)
+		"light": lighten(#FF6600, 10%),
+		"lighter": lighten(#FF6600, 20%),
+		"lightest": lighten(#FF6600, 45%)
+	),
+	"poplee": (
```
Comment (lucienbertin) [inline packages/scss/src/theming/_palettes.scss]: no palette figgo/poplee in the official release thx

Comment (jeremie-lucca) [reply packages/scss/src/theming/_palettes.scss]: It's only for the review.
I'll remove them when @Clotilde will be ok with the new version of these new versions.

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://358-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://358-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://358-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://358-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://358-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://358-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://358-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://358-pr-lucca-front-LuccaSA.surge.sh

File: packages/scss/src/components/_link.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+		color: _color("text.link", "hover");
+	}
+}
+
+.link {
+	background: transparent;
+	border: 0;
+	color: _color("text.link");
+	cursor: pointer;
+	text-decoration: underline;
```
[the remark this answers is not in the document]
Comment (jeremie-lucca) [reply packages/scss/src/components/_link.scss]: Yep, it should be an inline element

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://358-pr-lucca-front-LuccaSA.surge.sh

**Pull request merged**